karbon/karbon_api.go

```diff
@@ -22,6 +24,69 @@ type Client struct {
 	Meta            MetaService
 }
 
+type ClientService interface {
```

Comment:
The interface only seems to cover registration and should be named accordingly, imo.